pages/blog/conference-2023.md

```diff
@@ -0,0 +1,158 @@
+<img src='/img/posts/2023-summary/sngular.png' alt='sngular' width='500px' /> 
+</a>
+
+
+
+<a href='https://www.postman.com/' target='_blank'>
+<img src='/img/sponsors/postman.png' alt='postman' width='500px' />
+</a>
+
+<a href='https://www.apidays.global/' target='_blank'>
+<img src='/img/posts/2023-summary/apidays-logo-2023.png' alt='apidays' width='500px' />
+</a>
+
+We are truly grateful to them for their contributions!
+
+## Speakers
+
+What is a conference without our star speakers? These individuals took time out of their busy schedules to share their expertise. 
+They provided valuable industry insights, and we are proud of the network we built during the events.
+
+<Figure
+  src="/img/posts/2023-summary/speaker-2023.webp"
+/>
+
+
+You can also catch up on the live recording from [AACoT'23 Madrid](https://www.youtube.com/playlist?list=PLbi1gRlP7pihD-7XYb6c9wcTdkavDzF3_) and [AACoT'23 Bangalore](https://www.youtube.com/playlist?list=PLbi1gRlP7pijVocLZS7FeWKA4NBzJa7_Z).
+
+## Special Shout-Out
+
+I want to express my gratitude to these individuals and many others for their unwavering dedication in assisting with AACoT'23 behind the scenes. 
+They contributed to the design of website prototypes, conference design implementation, creating promotional videos, reviewing Call for Papers (CfPs), editing conference recordings, and providing constant communication and support. I am genuinely thankful for their hard work and commitment.
+
+<Profile profiles={[
+{
+    name: 'Aishat Muibudeen',
+    avatar: 'https://avatars.githubusercontent.com/Mayaleeeee',
+    link: 'https://github.com/Mayaleeeee'
+  }, 
+  {
+    name: 'Lukasz Gornicki',
+    avatar: 'https://avatars.githubusercontent.com/derberg',
+    link: 'https://github.com/derberg'
+  },
+  {
+    name: 'Azeez Elegbede',
+    avatar: 'https://avatars.githubusercontent.com/acethecreator',
+    link: 'https://github.com/acethecreator'
+  },
+  {
+    name: 'Barbanio González',
+    avatar: 'https://avatars.githubusercontent.com/Barbanio',
+    link: 'https://github.com/Barbanio'
+  },
+  {
+    name: 'Iván García Sainz-Aja',
+    avatar: 'https://avatars.githubusercontent.com/ivangsa',
+    link: 'https://github.com/ivangsa'
+  },
+  {
+    name: 'Dale Lane',
+    avatar: 'https://avatars.githubusercontent.com/dalelane',
+    link: 'https://github.com/dalelane'
+  },
+  {
+    name: 'Souvik De',
+    avatar: 'https://avatars.githubusercontent.com/Souvikns',
+    link: 'https://github.com/Souvikns'
+  },
+  {
+    name: 'Karuna Tata',
+    avatar: 'https://avatars.githubusercontent.com/starlightknown',
+    link: 'https://github.com/starlightknown'
+  }, 
+    {
+    name: 'Jonas Lagoni',
+    avatar: 'https://avatars.githubusercontent.com/jonaslagoni',
+    link: 'https://github.com/jonaslagoni'
+  },
+  {
+    name: 'Florence Njeri',
+    avatar: 'https://avatars.githubusercontent.com/Florence-Njeri',
+    link: 'https://github.com/Florence-Njeri'
+  }
+]} />
+
+## Where Too Next?
```

Comment:
Hi @thulieblack.
I noticed a small typo in the phrase "Where Too Next?" The correct usage should be "where to next?" with "to" instead of "too."

Reply:
missed that  thank you @omidNM24